el-debate/src/main/java/pl/elpassion/elspace/debate/DebatesRepository.kt

```diff
@@ -5,25 +5,30 @@ import com.elpassion.android.commons.sharedpreferences.createSharedPrefs
 import com.elpassion.sharedpreferences.gsonadapter.gsonConverterAdapter
```

Comment:
Find place where this method is being used and then explain why should I invert this boolean.